requirements.txt

```diff
@@ -1,14 +1,14 @@
-aniso8601==7.0.0
-Django==2.2.3
+aniso8601==6.0.0
+Django==2.2.4
 django-cors-headers==3.0.2
-graphene==2.1.6
-graphene-django==2.3.2
-graphql-core==2.2
-graphql-relay==0.4.5
+graphene==2.1.7
+graphene-django==2.4.0
+graphql-core==2.2.1
+graphql-relay==2.0.0
 Pillow==6.1.0
 promise==2.2.1
 psycopg2-binary==2.8.3
-pytz==2019.1
+pytz==2019.2
```

Comment:
Medium severity flaws identified in some dependencies - no exploitable conditions in code written so far